test/unit/vdom/elements.js

```diff
@@ -0,0 +1,34 @@
+import element from '../../lib/element';
+import { symbols, vdom } from '../../../src/index';
+
+const shadowDomV0 = !!document.createElement('div').createShadowRoot;
```

Comment:
We could check for v1 support first. That might be saner in the long run.

> And/or make a polyfill in namedslots which transforms v0 to v1

Can we do that easily?